string id
string[] item_list
string[] value_list # 
---
string res